client/apps/settings/client/curations/gucci/components/series.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import ImageUpload from 'client/apps/edit/components/admin/components/image_upload.coffee'
+import Paragraph from 'client/components/rich_text2/components/paragraph.coffee'
+
+export const SeriesAdmin = (props) => {
+  const { curation, onChange } = props
+
+  return (
+    <div className='admin-form-container series'>
+      <Row>
+        <Col lg={8}>
+          <div className='field-group'>
+            <label>About the Series</label>
+            <div className='bordered-input'>
+              <Paragraph
+                html={curation.get('about') || ''}
+                linked
+                layout='classic'
```

Comment:
Interesting, could it be worth adding a default layout on the `Paragraph` component? It seems like this could get confusing down the line, like the concept of 'classic' doesn't really apply to this Gucci curation. Not necessary, but on instinct this feels a little hacky -- maybe it just needs to be renamed?

Reply:
This just changes the font sizes on the component in the UI -- which defaults to the editorial font-size.  Could just change the default size as well to be classic.